crates/stackable-operator/src/commons/listener.rs

```diff
@@ -50,16 +50,19 @@ use crate::builder::pod::volume::ListenerOperatorVolumeSourceBuilder;
 )]
 #[serde(rename_all = "camelCase")]
 pub struct ListenerClassSpec {
-    pub service_type: ServiceType,
+    pub service_type: KubernetesServiceType,
 
     /// Annotations that should be added to the Service object.
     #[serde(default)]
     pub service_annotations: BTreeMap<String, String>,
 }
 
 /// The method used to access the services.
-#[derive(Serialize, Deserialize, Clone, Copy, Debug, JsonSchema, PartialEq, Eq)]
-pub enum ServiceType {
+//
+// Please note that this represents a Kubernetes type, so the name of the enum variant needs to exactly match the
+// Kubernetes service type.
+#[derive(Serialize, Deserialize, Clone, Copy, Debug, JsonSchema, PartialEq, Eq, strum::Display)]
+pub enum KubernetesServiceType {
```

Comment:
**style** (non-blocker): spaces between variants/fields with comments/attributes.

_left it as a non-blocker, since it is existing code_

Reply:
Fixed in 5e6453795b9945d7e3fd8b54fa7f9a6ab0442bc5